### PR TITLE
DPE-2425 Update version of mysql in the upgrade dependencies file

### PR DESCRIPTION
**Pull request by shayancanonical**
## Issue
When upgrading to `charmed-mysql` snap revision 69 (v8.0.34), we missed updating the upgrade dependencies

## Solution
Update the upgrade dependencies

### CODE REVIEW

File: src/dependency.json [hunk 1]
```diff
@@ -3,12 +3,12 @@
     "dependencies": {},
     "name": "mysql",
     "upgrade_supported": ">0",
-    "version": "1"
+    "version": "2"
   },
   "snap": {
     "dependencies": {},
     "name": "charmed-mysql",
     "upgrade_supported": "~8.0.31",
```
[the remark this answers is not in the document]
Comment (paulomach) [reply src/dependency.json:11]: as long as there's not a breaking change, `upgrade_supported` shouldn't be changed.

**Review by paulomach — APPROVED** [review]

File: src/dependency.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (taurus-forever) [reply src/dependency.json:11]: +1 to Paulo.

[the remark this answers is not in the document]
Comment (taurus-forever) [reply src/dependency.json:11]: BTW, should we start Semantic versioning (as discussed today)?
1 == 1.0.0 -> 1.0.1

[the remark this answers is not in the document]
Comment (shayancanonical) [reply src/dependency.json:11]: I think that semantic versioning would be ideal, but cant wrap my head around any fallout from switching from numeric versioning to semantic. Would you know @paulomach?

[the remark this answers is not in the document]
Comment (taurus-forever) [reply src/dependency.json:11]: IF you commit 1.0.1 today, you can commit 2 tomorrow.
IF you commit 2 today, you cannot return to 1... which is not mission critical, we can start counting from 42 without any issues.

**Review by taurus-forever — APPROVED** [review]

**Comment by shayancanonical** [comment]
Closing since we do not want to update the charm version, and the mysqld version is already `8.0.34` in main

**Pull request closed without merging**